v4-client-js/src/clients/modules/post.ts

```diff
@@ -751,15 +751,13 @@ export class Post {
       subaccount.wallet,
       () => Promise.resolve([msg]),
       false,
-      undefined,
+      this.defaultDydxGasPrice,
```

Comment:
Oh this should be USDC gas

Reply:
oh huh I thought claiming rewards would be in the native token - will update!